### PR TITLE
fix(drain): default index-prefix for elasticsearch drain should be null

### PR DESCRIPTION
**Pull request by KannarFr**

### CODE REVIEW

File: bin/clever.js [hunk 1]
```diff
@@ -417,8 +417,7 @@ function run () {
     drainIndexPrefix: cliparse.option('index-prefix', {
       aliases: ['i'],
       metavar: 'index_prefix',
-      description: '(ElasticSearch drains) Index prefix',
-      default: 'logstash-<YYYY-MM-DD>',
+      description: '(ElasticSearch drains) Index prefix default to `logstash-<YYYY-MM-DD>`',
```
[the remark this answers is not in the document]
Comment (KannarFr) [reply bin/clever.js]: applied.

**Comment by KannarFr** [comment]
@davlgd tested and validated. I just added `custom` in the drains list: https://github.com/CleverCloud/clever-tools/pull/815/files#diff-0a728e690c0da8df121ce69b7dd6c676fbcfdd43a69222fb2f5e49dedbd20f79R46.

**Comment by davlgd** [comment]
> @davlgd tested and validated. I just added `custom` in the drains list: https://github.com/CleverCloud/clever-tools/pull/815/files#diff-0a728e690c0da8df121ce69b7dd6c676fbcfdd43a69222fb2f5e49dedbd20f79R46.

LGTM, maybe we should remove the `<` and `>` to avoid confusion ?

**Comment by KannarFr** [comment]
> LGTM, maybe we should remove the `<` and `>` to avoid confusion ?

Not sure it's a drop-in, up to you.

**Comment by hsablonniere** [comment]
FYI: after a discussion with @KannarFr and @davlgd, I removed the `<` and `>`.

**Review by hsablonniere — APPROVED** [review]

**Pull request merged**